completable-reactor-runtime/src/main/kotlin/ru/fix/completable/reactor/runtime/execution/ExecutionBuilder.kt

```diff
@@ -344,6 +353,20 @@ class ExecutionBuilder(
                 processingVertices.values)
     }
 
+    /**
+     * Check if transition is marked as `onElse`
+     * and merge results are not satisfied by another existing transitions
+     */
+    private fun isOnElse(
+            transition: GlTransition,
+            mergerablePvx: ProcessingVertex,
+            context: ExecutionBuilder.MergePayloadContext
+    ): Boolean {
+        return transition.isOnElse &&
+                mergerablePvx.outgoingTransitions.all { !it.mergeStatuses.contains(context.mergeResult) }
```

Comment:
`mergerablePvx.outgoingTransitions.all` includes current transition itself.   
mergeResult could be NULL.  
Maybe it would be better to compare mergeResult with all statuses except one with enabled isOnElse flag ?